Core/clim-core/frames.lisp

```diff
@@ -1630,10 +1630,12 @@ have a `pointer-documentation-pane' as pointer documentation,
 		   do (return-from find-dest-translator translator))
 	     nil)
 	   (do-feedback (window x y state)
-	     (funcall feedback-fn frame from-presentation window
-			initial-x initial-y x y state))
+	     (when (typep window 'clim-stream-pane)
```

Comment:
it solves immediate issue for me, but are we allowed to limit d&d to only clim-stream-pane? Also, why does it break, if window is something different than clim-stream-pane?

Reply:
I think is not specified; the problem is that the function frame-drag-and-drop-feedback try to draw in the stream, and this invoke INVOKE-WITH-SHEET-MEDIUM-BOUND; but there is no method defined for all classes of sheets.

Reply:
in that case please specialize `frame-drag-and-drop-feedback` stream argument on `output-stream class ` and add new method (above the existing one), which is not specialized on anything and is a no-op (empty body).

We'd want to specialize on `output-recording-stream`, but we have take into account also that we have `encapsulating-stream`, which isn't recording (but underlaying stream may be output-recording-stream).

Reply:
also please squash changes into single commit when you do